backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/picking/api/impl/PackagingDAO.java

```diff
@@ -133,7 +134,29 @@ private IQuery<I_M_Packageable_V> createQuery(@NonNull final PackageableQuery qu
 		// Filter: PreparationDate
 		if (query.getPreparationDate() != null)
 		{
-			queryBuilder.addEqualsFilter(I_M_Packageable_V.COLUMN_PreparationDate, query.getPreparationDate(), DateTruncQueryFilterModifier.DAY);
+			queryBuilder.addEqualsFilter(I_M_Packageable_V.COLUMNNAME_PreparationDate, query.getPreparationDate(), DateTruncQueryFilterModifier.DAY);
+		}
+
+		//
+		// Filter: IsProvisioningFixedDate
+		if (query.getMaximumPreparationDate() != null)
+		{
+			queryBuilder.addFilter(queryBL.createCompositeQueryFilter(I_M_Packageable_V.class)
+					.setJoinOr()
+					.addCompareFilter(I_M_Packageable_V.COLUMNNAME_DatePromised, CompareQueryFilter.Operator.LESS_OR_EQUAL, query.getMaximumPreparationDate().toInstant())
```

Comment:
Something is weird
* COLUMNNAME_DatePromised
* query.getMaximumPreparationDate()

Reply:
done